src/frontends/tensorflow_common/src/openvino_conversions.cpp

```diff
@@ -53,6 +53,13 @@ std::shared_ptr<ov::opset8::Transpose> make_transpose(const ov::Output<ov::Node>
     return transpose;
 }
 
+std::shared_ptr<ov::opset8::Reshape> make_reshape(const ov::Output<ov::Node>& arg,
+                                                  const std::vector<int64_t>& new_shape) {
+    auto new_shape_node = std::make_shared<ov::opset8::Constant>(element::i64, Shape{new_shape.size()}, new_shape);
+    auto reshape = std::make_shared<ov::opset8::Reshape>(arg, new_shape_node, true);
+    return reshape;
+}
```

Comment:
can we put all this stuff into `tensorflow_common/src/utils.cpp` and remove `openvino_conversions`?
upto you, but I am going to do this because `openvino_conversions` looks weird:)